cfep-26.md

```diff
@@ -0,0 +1,60 @@
+<tr><td> Created </td><td> Jun 13, 2025</td></tr>
+<tr><td> Updated </td><td> Jun 13, 2025</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> link to the PR for the implementation, NA if not availble </td></tr>
+</table>
+
+## Abstract
+
+This CFEP proposes a policy for how to spend unrestricted conda-forge project funds.
+It aims to provide a set of transparent principles and guidelines for decision-making
+around the transparent use of administration of these funds, ensuring that they are used
+in a way that benefits the conda-forge community as a whole.
+
+## Motivation
+
+conda-forge is a NumFOCUS (Austin, TX based non-profit 501(c)(3)) fiscally sponsored project,
+as such the conda-forge project has to align with accounting and legal regulations for US-based non-profits.
+Under these considerations, nonprofit donations are classified into two categories:
+**unrestricted funds** which may be used for any legal purpose appropriate to the
+organization and **restricted funds**, monies set aside for a particular purpose or
+program (for example project-specific deliverables, educational program, etc.).
+
+Up to now, conda-forge remains a largely volunteer-run project without significant
+amounts of unrestricted funds (either from individual donations or corporate sponsorships),
+thus robust policies for spending these funds have not yet been critical for the project.
+However, as the conda-forge project grows there is also the potential for increased
+donations and grants going directly to the project.
+Therefore, it is important to establish clear, transparent, and fair principles for how
+unrestricted funds can be used to support the community and its goals. Additionally,
+having such principles and processes in place will make it easier to attract and manage
+donations and other types of contributions moving forward.
+
+## Scope
+
+The following items are in scope for this CFEP:
+
+- Principles of spending project funds (what to fund, and who to pay for this).
+- Processes for decision-making around spending project funds and fund management.
+- Guidelines to manage conda-forge project funds.
+
+Out of scope items:
+
+- Decision-making around spending project funds on a specific project or activity.
+- Principles and processes for spending restricted funds, which are earmarked for
+  specific purposes or projects.
+  This includes most of the grant funding, which is usually earmarked for certain
+  activities/deliverables and goes to an Institutional Partner rather than directly to
+  the conda-forge project, and companies or institutions funding specific features.
+  *For grants though, we expect the research/project leader and funded team to align their
+  work with the needs of conda-forge and be receptive to feedback from conda-forge
+  core team members, maintainers, and contributors.*
```

Comment:
I'm unsure if there is better wording or if these expectations are already set elsewhere, @jaimergp.